conda.recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dask
+  version: 'nightly'
+
+build:
+  number: 1
```

Comment:
Version numbering can be tricky.  I strongly recommend stealing most of this from odo